src/Geocoder/GeocoderResult.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Geocoder;
+
+/**
+ * This is the interface that is always return from a Geocoder.
+ *
+ * @author William Durand <william.durand1@gmail.com>
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+interface GeocoderResult extends \IteratorAggregate, \Countable
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function getIterator();
+
+    /**
+     * {@inheritDoc}
+     */
+    public function count();
```

Comment:
This methods can be removed right? The method is already covered by the interfaces.